src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import core.basesyntax.model.enums.Color;
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private final Integer randomNumber = 20;
+    private final Integer defaulSize = 10;
+    private final Integer fiduresCount = 5;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
```

Comment:
should be class-level `private final` field, no need to create each time when we call this method